packages/core/hardhat.config.ts

```diff
@@ -167,7 +77,7 @@ task('deploy-system')
         )
       }
       const provider = new SphinxJsonRpcProvider(hre.network.config.url)
-      const signer = await hre.ethers.provider.getSigner()
+      const signer = new Wallet(process.env.PRIVATE_KEY!).connect(provider)
```

Comment:
This is necessary because sending transactions on OKX Chain only works with the SphinxJsonRPCProvider. So we can't just use the signer provided by the HRE here.